nixos/modules/virtualisation/anbox.nix

```
@@ -25,6 +29,28 @@ let
     };
   };
 
+  finalImage = if cfg.imageModifications == "" then cfg.image else ( pkgs.callPackage (
+    { runCommandNoCC, squashfsTools }:
+
+    runCommandNoCC "${cfg.image.name}-modified.img" {
+      nativeBuildInputs = [
+        squashfsTools
+      ];
+    } ''
+      echo "→ Extracting Anbox root image..."
```

```suggestion
      echo "-> Extracting Anbox root image..."
```
we shouldn't use special unicode characters